### DISCUSSION TITLE
Is there a way to show key presses from showing in the top right?

### DISCUSSION THREAD

**Discussion by joedborg** [blinksh/blink#1921] [Support] [answered]
I can’t see an option in `config`.  I mostly use Blink on an iPad with keyboard so don’t need it and it shows passwords in plain text.

**Comment by carloscabanero** [comment, marked as answer]
Hi! It sounds like you may have the KeyCast feature enabled. Is that it? If not, please send a screenshot of what you are seeing. Thanks!

<img width="697" alt="image" src="https://github.com/blinksh/blink/assets/536729/ba8cd2ac-811f-4c1e-aa0a-5696eb84fe0c">

**Comment by joedborg** [comment]
Ah thanks, yes.  I had already tried that but seem you have to restart Blink, at least in my case, for it to take effect.
